grails-app/views/admin/showSettings.gsp

```diff
@@ -1,4 +1,4 @@
-<%@ page import="org.pih.warehouse.core.RoleType" %>
+<%@ page import="org.pih.warehouse.LocalizationUtil; org.pih.warehouse.core.RoleType" %>
```

Comment:
nitpicky: I know it was done automaticallty, but we used to separate the imports in gsp templates to be one import per line

Reply:
Yeah this is my laziness since I was doing the automatic import and noticed it while creating the PR it's in one line, I'll fix that